assets/javascripts/kitten/helpers/utils/unique-id.test.js

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai'
+import { uniqueIdUtils } from 'kitten/helpers/utils/unique-id'
+
+describe('getNextId()', () => {
+  it('get next id', () => {
```

Comment:
Est-ce qu'on peut avoir un test plus descriptif ici qui dit ce qu'on attend de cette méthode dans ce test ? Par exemple `it('returns a string starting at 1', () => {`.